### PR TITLE
Profiles: ENS Registration Intro Screen

### PR DESCRIPTION
**Pull request by jxom**
This PR implements the ENS registration intro screen

https://www.loom.com/share/08af08e45096415a9e88e2af983b9fd3

### CODE REVIEW

File: src/components/ens-registration/IntroMarquee/IntroMarquee.tsx [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+}: {
+  name: string;
+  onPress: () => void;
+  onPressCancel: () => void;
+  onPressStart: () => void;
+}) {
+  return (
+    <ButtonPressAnimation
+      onCancel={({ nativeEvent: { state, close } }: any) => {
+        if (state === 5 && close) {
```
Comment (estebanmino) [inline src/components/ens-registration/IntroMarquee/IntroMarquee.tsx:71]: could you add a comment there to explain the state === 5 ?

File: src/hooks/useLoadGlobalLateData.js [hunk 2]
```diff
@@ -27,10 +31,11 @@ export default function useLoadGlobalLateData() {
     const promises = [];
     const p1 = loadWalletBalanceNamesToCache();
     const p2 = dispatch(nonceManagerLoadState());
-    promises.push(p1, p2);
+    const p3 = prefetchAccountENSDomains({ accountAddress });
+    promises.push(p1, p2, p3);
```
Comment (estebanmino) [inline src/hooks/useLoadGlobalLateData.js:35]: not related to this pr but I feel we can resolve and save ENS name of the wallet here as well

File: src/screens/ENSIntroSheet.tsx [hunk 3]
```diff
@@ -0,0 +1,235 @@
...
+                  )}
+                  {isSuccess && (
+                    <>
+                      {domains?.length === 0 ? (
+                        <Button
+                          backgroundColor={colors.appleBlue}
+                          onPress={handleNavigateToSearch}
+                          textProps={{ weight: 'heavy' }}
+                        >
+                          􀠎 Find your name
```
Comment (estebanmino) [inline src/screens/ENSIntroSheet.tsx:143]: ups lang.t missing

File: src/screens/ENSIntroSheet.tsx [hunk 4]
```diff
@@ -0,0 +1,235 @@
...
+                        </Button>
+                      ) : (
+                        <Stack space="15px">
+                          {domains?.length === 1 ? (
+                            <Button
+                              backgroundColor={colors.appleBlue}
+                              onPress={handleSelectExistingName}
+                              textProps={{ weight: 'heavy' }}
+                            >
+                              Use {domains[0].name}
```
Comment (estebanmino) [inline src/screens/ENSIntroSheet.tsx:153]: ups lang.t missing

File: src/screens/ENSIntroSheet.tsx [hunk 5]
```diff
@@ -0,0 +1,235 @@
...
+                            </Button>
+                          ) : (
+                            <Button
+                              backgroundColor={colors.appleBlue}
+                              onPress={handleSelectExistingName}
+                              textProps={{ weight: 'heavy' }}
+                            >
+                              Use an existing ENS name
```
Comment (estebanmino) [inline src/screens/ENSIntroSheet.tsx:161]: ups lang.t missing

File: src/screens/ENSIntroSheet.tsx [hunk 6]
```diff
@@ -0,0 +1,235 @@
...
+                            </Button>
+                          )}
+                          <Button
+                            backgroundColor={colors.transparent}
+                            borderColor={colors.transparent}
+                            color={colors.appleBlue}
+                            onPress={handleNavigateToSearch}
+                            textProps={{ size: 'lmedium', weight: 'heavy' }}
+                          >
+                            Search for a new ENS name
```
Comment (estebanmino) [inline src/screens/ENSIntroSheet.tsx:171]: ups lang.t missing

File: src/screens/SelectENSSheet.tsx [hunk 7]
```diff
@@ -0,0 +1,129 @@
...
+    [accentColor, handleSelectENS]
+  );
+
+  return (
+    // @ts-expect-error JavaScript component
+    <Sheet>
+      <Inset horizontal="19px" top="10px">
+        <Stack space="15px">
+          <Heading align="center" size="18px">
+            Select ENS Name
```
Comment (estebanmino) [inline src/screens/SelectENSSheet.tsx:108]: ups lang.t missing

**Comment by estebanmino** [comment]
not sure what happened but this profile picture is blue 
image

**Review by estebanmino — APPROVED** [review]
looking 💯  on android and ios

**Comment by jxom** [comment]
@estebanmino – added some preloading for the marquee images 🙏

**Review by estebanmino — APPROVED** [review]
🚀

**Pull request merged**